grpc/private/routing.go

```diff
@@ -0,0 +1,108 @@
+package private
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/keratin/authn-server/api"
+	"github.com/keratin/authn-server/grpc/public"
+	"github.com/keratin/authn-server/lib/route"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+/*
+func router(app *api.App) http.Handler {
+	r := mux.NewRouter()
```

Comment:
I used this comment block while writing this file to constantly see what routes I should check and add. It should be removed or add a comment referencing where these came from.